lxd/containers.go

```diff
@@ -213,7 +213,7 @@ func containersRestart(d *Daemon) error {
 }
 
 func containersShutdown(d *Daemon) error {
-	results, err := d.ListRegularContainers()
+	results, err := dbContainersList(d.db, cTypeRegular)
```

Comment:
Oops, for some reason I thought I was doing commit by commit review, not the whole diff...